tycho-integration-test/src/main.rs

```diff
@@ -519,7 +519,7 @@ async fn process_state(
                     amount
                 }
                 Err((e, state_overwrites, metadata)) => {
-                    let error_msg = e.to_string();
+                    let error_msg = format_error_chain(&e);
```

Comment:
I think this will mess up the logic that comes afterwards (getting then reason and error name). Can we not do this here please? Here it is already correct

Reply:
I tested it and it seemed to be working fine, but I'll revert it for now.